sig-aws/2017_goals.md

```diff
@@ -0,0 +1,26 @@
+# Kubernetes sig-aws Goals (2017)
+
+# IAM improvements
+ * We hope to offer a more seamless integration path to taking advantage of IAM roles in Kubernetes
+ * We are looking at implementations such as [kube2iam](https://github.com/jtblin/kube2iam)
+
+##### Statement
+
+Kubernetes components now have the ability to take advantage of IAM roles. The process is intuitive and easy.
+
+# Networking in AWS
+ * We hope to offer a knowledge base for networking in Kubernetes on AWS
+ * We will consolidate networking reports and metrics around Kubernetes e2e tests and grow our metric list over time
+
+##### Statement
+
+A user can go to a single public location, and see a side by side comparison of networking avenues in Kubernetes on AWS.
+
+
+# Federation
+
+* We need to improve and harden our federation support in AWS
```

Comment:
@justinsb you mentioned a few more on our last sig call.  

@kris-nova how can I assist?  The notes are in the last meeting.  Should we review this next sig?